appeals/web/src/server/lib/mappers/appeal/submappers/ip-comments.mapper.js

```diff
@@ -0,0 +1,27 @@
+import { mapDocumentStatus } from '#lib/display-page-formatter.js';
+
+/** @type {import('../appeal.mapper.js').SubMapper} */
+export const mapIpComments = ({ appealDetails, currentRoute }) => {
+	return {
+		id: 'ip-comments',
+		display: {
+			tableItem: [
+				{
+					text: 'Interested party comments'
+				},
+				{
+					text: mapDocumentStatus(appealDetails?.documentationSummary?.ipComments?.status)
+				},
+				{
+					text: 'Not applicable'
+				},
+				{
+					html:
```

Comment:
I think something like this would be a bit more readable:

```
const linkLabel = appealDetails?.documentationSummary?.ipComments?.status === 'received' ? 'Review' : 'Add';
...
html: `<a href="${currentRoute}/interested-party-comments" data-cy="review-ip-comments" class="govuk-link">${linkLabel} <span class="govuk-visually-hidden">I P comments</span></a>`;
```

Reply:
fair point, better now?